itests/testdata/issue_smart_asset_tx_testdata.go

```diff
@@ -0,0 +1,266 @@
+package testdata
+
+import (
+	f "github.com/wavesplatform/gowaves/itests/fixtures"
+	utl "github.com/wavesplatform/gowaves/itests/utilities"
+)
+
+const (
+	IssueSmartAssetMinVersion = 2
+)
+
+func GetPositiveAssetScriptData(suite *f.BaseSuite) map[string]IssueTestData[ExpectedValuesPositive] {
+	return map[string]IssueTestData[ExpectedValuesPositive]{
+		"Valid script, true as expression": *NewIssueTestData(
+			utl.GetAccount(suite, utl.DefaultSenderNotMiner),
+			utl.RandStringBytes(5, utl.CommonSymbolSet),
+			utl.RandStringBytes(20, utl.CommonSymbolSet),
+			100000000000,
+			8,
+			true,
+			utl.GetScriptBytes(suite, "BQbtKNoM"),
+			utl.MinIssueFeeWaves,
+			utl.GetCurrentTimestampInMs(),
+			utl.TestChainID,
+			ExpectedValuesPositive{
+				WavesDiffBalance: utl.MinIssueFeeWaves,
+				AssetBalance:     100000000000,
+			}),
+		"Valid script, size 8192 bytes": *NewIssueTestData(
+			utl.GetAccount(suite, utl.DefaultSenderNotMiner),
+			utl.RandStringBytes(5, utl.CommonSymbolSet),
+			utl.RandStringBytes(20, utl.CommonSymbolSet),
+			100000000000,
+			8,
+			true,
+			utl.GetScriptBytes(suite, "BgEEA3N0cgI3VGhpcyB0ZXh0IGlzIG5lY2Vzc2FyeSB0byBnZXQgdGhlIHNjcmlwdCBv"+
```

Comment:
Why not to use one very long line? As in other tests with a big compiled scripts.

Reply:
Or you can put the script in file. We already have function to read such files (pkg/state/invoke_applier_test.go:92).

Reply:
Was added new folder with scripts and some methods for work with it.